docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,299 @@
+```
+
+</div>
+<div class="tab-pane fade" id="install-tab-helm2" markdown="1">
+Use Helm 2 to install the latest official `alpha` release of Crossplane, suitable for community use and testing:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+</div>
+<div class="tab-pane fade" id="install-tab-helm3-master" markdown="1">
+Use Helm 3 to install the latest `master` pre-release version of Crossplane:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-master https://charts.crossplane.io/master/
+helm search repo crossplane-master --devel
+
+# Kubernetes 1.15 and newer versions
+helm install crossplane --namespace crossplane-system crossplane-master/crossplane --version <version> --devel
+```
+
+For example:
+```
+helm install crossplane --namespace crossplane-system crossplane-master/crossplane --version 0.11.0-rc.100.gbc5d311 --devel
+```
+
+</div>
+<div class="tab-pane fade" id="install-tab-helm2-master" markdown="1">
+Use Helm 2 to install the latest `master` pre-release version of Crossplane, which is suitable for testing pre-release versions:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-master https://charts.crossplane.io/master/
+helm search crossplane-master --devel
+
+helm install --name crossplane --namespace crossplane-system crossplane-master/crossplane --version <version>
+```
+
+For example:
+
+```
+helm install --name crossplane --namespace crossplane-system crossplane-master/crossplane --version 0.11.0-rc.100.gbc5d311
+```
+
+</div>
+</div>
+
+## Install Crossplane CLI
+The [Crossplane CLI] adds a set of `kubectl crossplane` commands to simplify common tasks:
+```
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/master/bootstrap.sh | bash
+```
+
+## Select Provider
+Install and configure a provider for Crossplane to use for infrastructure provisioning:
+<ul class="nav nav-tabs">
+<li class="active"><a href="#provider-tab-aws" data-toggle="tab">AWS</a></li>
+<li><a href="#provider-tab-gcp" data-toggle="tab">GCP</a></li>
+<li><a href="#provider-tab-azure" data-toggle="tab">Azure</a></li>
+<li><a href="#provider-tab-alibaba" data-toggle="tab">Alibaba</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="provider-tab-aws" markdown="1">
+
+### Install AWS Provider
+
+```
+PACKAGE=crossplane/provider-aws:master
+NAME=provider-aws
+
+kubectl crossplane package install --cluster --namespace crossplane-system ${PACKAGE} ${NAME}
+```
+
+### Get AWS Account Keyfile
+
+Using an AWS account with permissions to manage RDS databases:
+```
+AWS_PROFILE=default && echo -e "[default]\naws_access_key_id = $(aws configure get aws_access_key_id --profile $AWS_PROFILE)\naws_secret_access_key = $(aws configure get aws_secret_access_key --profile $AWS_PROFILE)" > creds.conf
```

Comment:
Could we guide the user to create a new access key via `aws iam create-access-key` instead? Leading a new user into pushing their current credentials into a (possibly non-local) cluster feels like a potential `record-needle-scratch.wav` moment.

Reply:
yes that's a better way IMO, but suggest we do that with a follow-on PR

Reply:
although IIRC they limit you to 2 access keys, so wouldn't want to force that:
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-limits.html